### PR TITLE
[WIP] move galaxy docs to a separate landing page.

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Move galaxy docs to a separate build. This allows us to version ansible docs separate from galaxy docs, while still keeping the same docsite theme etc.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs.ansible.com
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/galaxy/docsite/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+Homepage and Documentation Source for Ansible Galaxy
+====================================================
+
+This project hosts the source behind [docs.ansible.com/galaxy](https://docs.ansible.com/galaxy)
```
Comment (acozine) [inline docs/galaxy/docsite/README.md:4]:
```suggestion
This project hosts the source behind [docs.ansible.com/ansible-galaxy](https://docs.ansible.com/ansible-galaxy)
```

File: docs/galaxy/docsite/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Contributions to the documentation are welcome. To make changes, submit a pull request that changes the reStructuredText files in the `rst/` directory only, and the core team can do a docs build and push the static files.
+
+If you wish to verify output from the markup such as link references, you may install sphinx and build the documentation by running `make webdocs` from the `ansible/docsgalaxy/docsite` directory.
```
Comment (acozine) [inline docs/galaxy/docsite/README.md:8]: I know you just copied this from the main doc, so changes should probably go there . . . but I'm noting them down while I'm thinking of them. 

Can we add a link to Sphinx to the words "install sphinx"?
Also, there's a slash missing in the directory path (docsgalaxy).

File: docs/galaxy/docsite/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (acozine) [inline docs/galaxy/docsite/README.md:9]: Actually, we could just link to the page on docs.ansible.com, and not have to maintain this stuff in multiple places.

File: docs/galaxy/docsite/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+To include module documentation you'll need to run `make webdocs` at the top level of the repository. The generated html files are in `docsite/htmlout/`.
+
+To limit module documentation building to a specific module, run `MODULES=NAME make webdocs` instead. This should make testing module documentation syntax much faster. Instead of a single module, you can also specify a comma-separated list of modules. In order to skip building documentation for all modules, specify non-existing module name, for example `MODULES=none make webdocs`.
+
+If you do not want to learn the reStructuredText format, you can also [file issues] about documentation problems on the Ansible GitHub project.
+
+To install sphinx and the required theme, install ``pip`` and then ``pip install sphinx sphinx_rtd_theme``
+
+[file issues]: https://github.com/ansible/ansible/issues
```
Comment (acozine) [inline docs/galaxy/docsite/README.md:18]: A link to https://github.com/ansible/ansible/issues/new?template=documentation_report.md would be quicker.

File: docs/galaxy/docsite/rst/404.rst [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+*****
+Oops!
+*****
+
+The version of the Galaxy documentation you were looking at doesn't contain that page.
+
+.. image:: images/cow.png
+   :alt: Cowsay 404
+
+Use the back button to return to the version you were browsing, or use the navigation at left to explore our latest release. Once you're on a non-404 page, you can use the version-changer to select a version.
```
Comment (acozine) [inline docs/galaxy/docsite/rst/404.rst:12]: Do we need to update the docsite repo too, to make sure the `ansible-galaxy` directory does the right thing for 404s?

File: docs/galaxy/docsite/rst/conf.py [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+# Options for HTML output
+# -----------------------
+
+html_theme_path = ['../_themes']
+html_theme = 'sphinx_rtd_theme'
+html_short_title = 'Galaxy Documentation'
+html_show_sphinx = False
+
+html_theme_options = {
+    'canonical_url': "https://docs.ansible.com/ansible/latest/",
```
Comment (acozine) [inline docs/galaxy/docsite/rst/conf.py:137]: Should this point to `/ansible-galaxy/` for this section of the docs?

File: docs/galaxy/docsite/rst/conf.py [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+# If true, the index is split into individual pages for each letter.
+# html_split_index = False
+
+# If true, the reST sources are included in the HTML build as _sources/<name>.
+html_copy_source = False
+
+# If true, an OpenSearch description file will be output, and all pages will
+# contain a <link> tag referring to it.  The value of this option must be the
+# base URL from which the finished HTML is served.
+html_use_opensearch = 'https://docs.ansible.com/ansible/latest'
```
Comment (acozine) [inline docs/galaxy/docsite/rst/conf.py:211]: Here too . . . `ansible-galaxy`?

File: docs/galaxy/docsite/rst/conf.py [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+
+# If false, no module index is generated.
+# latex_use_modindex = True
+
+autoclass_content = 'both'
+
+intersphinx_mapping = {'python': ('https://docs.python.org/2/', (None, '../python2-2.7.13.inv')),
+                       'python3': ('https://docs.python.org/3/', (None, '../python3-3.6.2.inv')),
+                       'jinja2': ('http://jinja.pocoo.org/docs/', (None, '../jinja2-2.9.7.inv')),
+                       'ansible': ('https://docs.ansible.com/ansible/devel/', (None, 'objects.inv'))}
```
Comment (acozine) [inline docs/galaxy/docsite/rst/conf.py:271]: Do we want `latest` here?

File: docs/galaxy/docsite/rst/user_guide/installing_collections.rst [hunk 9]
```diff
@@ -3,20 +3,21 @@
 Installing collections from Galaxy
 ==================================
 
-.. include:: ../../shared_snippets/installing_collections.txt
-
+.. include:: ../../../../docsite/rst/shared_snippets/installing_collections.txt
```
Comment (acozine) [inline docs/galaxy/docsite/rst/user_guide/installing_collections.rst:6]: So many dots!

**Review by acozine — COMMENTED** [review]
I'm jumping the gun here, since this PR is still WIP, but I wanted to record my Friday thoughts now, since I won't remember them on Monday otherwise.

**Comment by samccann** [comment]
closing for now. Will reopen when I get the chance to get back to this idea.

**Pull request closed without merging**